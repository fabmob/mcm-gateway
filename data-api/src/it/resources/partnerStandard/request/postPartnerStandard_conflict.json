{
  "standardName": "FNMS",
  "isActive": false,
  "versionStandard": "V1.2",
  "versionDataMapping": "V1.2",
  "partnerId": "6c317901-749f-4d7d-82bb-afdc737b26b3",
  "partnerActionsId": "5e71dcb3-ca60-4a2a-9896-c509ad1ca756",
  "adaptersId": "40142f60-9694-479f-a6cd-28b199b5e240"
}