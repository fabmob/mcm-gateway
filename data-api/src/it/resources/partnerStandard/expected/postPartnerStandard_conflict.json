{
  "callId": "FAKE_UUID",
  "errorCode": 409,
  "description": "The MSP with id FAKE_UUID does not handle this feature : PING",
  "label": "Conflict",
  "status": 409,
  "timestamp": "2080-01-01T00:00:00.154+01:00"
}